### ISSUE TITLE
Destructuring array cause error

### ISSUE THREAD

**Issue by arielschiavoni** [esdoc/esdoc#76]
Hi,

I get an error when I try to use array destructuring. I guess this bug is somehow related to #65.

Example:

```js
const [first] = [1, 2, 3]
```

```sh
$ esdoc -c esdoc.json
/Users/asc/Projects/2vizcon/gitlab/aschiavoni/devkit-server/node_modules/esdoc/out/src/Publisher/Builder/DocResolver.js:121
          if (autoPrivate && this.name.charAt(0) === '_') {
                                      ^

TypeError: Cannot read property 'charAt' of null
    at Object.<anonymous> (/Users/asc/Projects/2vizcon/gitlab/aschiavoni/devkit-server/node_modules/esdoc/out/src/Publisher/Builder/DocResolver.js:121:39)
    at /Users/asc/Projects/2vizcon/gitlab/aschiavoni/devkit-server/node_modules/esdoc/node_modules/taffydb/taffy.js:743:17
    at each (/Users/asc/Projects/2vizcon/gitlab/aschiavoni/devkit-server/node_modules/esdoc/node_modules/taffydb/taffy.js:126:17)
    at Object.<anonymous> (/Users/asc/Projects/2vizcon/gitlab/aschiavoni/devkit-server/node_modules/esdoc/node_modules/taffydb/taffy.js:740:7)
    at Object.API.(anonymous function) [as update] (/Users/asc/Projects/2vizcon/gitlab/aschiavoni/devkit-server/node_modules/esdoc/node_modules/taffydb/taffy.js:166:18)
    at DocResolver._resolveAccess (/Users/asc/Projects/2vizcon/gitlab/aschiavoni/devkit-server/node_modules/esdoc/out/src/Publisher/Builder/DocResolver.js:119:20)
    at DocResolver.resolve (/Users/asc/Projects/2vizcon/gitlab/aschiavoni/devkit-server/node_modules/esdoc/out/src/Publisher/Builder/DocResolver.js:51:12)
    at CoverageBuilder.DocBuilder (/Users/asc/Projects/2vizcon/gitlab/aschiavoni/devkit-server/node_modules/esdoc/out/src/Publisher/Builder/DocBuilder.js:57:42)
    at new CoverageBuilder (/Users/asc/Projects/2vizcon/gitlab/aschiavoni/devkit-server/node_modules/esdoc/out/src/Publisher/Builder/CoverageBuilder.js:33:81)
    at publish (/Users/asc/Projects/2vizcon/gitlab/aschiavoni/devkit-server/node_modules/esdoc/out/src/Publisher/publish.js:156:5)

```

**Comment by h13i32maru** [comment]
@arielschiavoni Hi! Thanks for this issue!
This is ESDoc bug :sweat: 
I will try to fix it, and please wait a little.

**Comment by h13i32maru** [comment]
I published v0.2.6 that fixed this issue. Please try to use it.
